src/main/java/pl/byczazagroda/trackexpensesappbackend/exception/GlobalExceptionHandlerController.java

```diff
@@ -47,7 +54,14 @@ public String handleMethodArgumentNotValidException(MethodArgumentNotValidExcept
     @ExceptionHandler(ResourceNotSavedException.class)
     @ResponseStatus(HttpStatus.INTERNAL_SERVER_ERROR)
     public String resourceNotSavedHandler(ResourceNotSavedException e) {
-        log.error(String.format("ResourceNotSavedException: %s" ,  e.getMessage()));
+        log.error(String.format("ResourceNotSavedException: %s", e.getMessage()));
+        return e.getMessage();
+    }
+
+    @ExceptionHandler(ResourceNotDeletedException.class)
+    @ResponseStatus(HttpStatus.NO_CONTENT)
```

Comment:
Why ``NO_CONTENT``? I'll discuss it with @dawciobiel later.

Reply:
I used NO_CONTENT status, because that was the requirement in the task:
"When Wallet with identifier doesn't exist status should be set to: 204 No Content"